packages/opentelemetry-semantic-conventions-ai/opentelemetry/semconv/ai/__init__.py

```diff
@@ -45,19 +45,25 @@ class SpanAttributes:
 
 
 class Events(Enum):
-    VECTOR_DB_QUERY_EMBEDDINGS = "vector_db.query.embeddings"
-    VECTOR_DB_QUERY_RESULT = "vector_db.query.result"
+    DB_QUERY_EMBEDDINGS = "db.query.embeddings"
+    DB_PINECONE_QUERY_RESULT = "db.pinecone.query.result"
```

Comment:
why not use the same name for both of them?

Reply:
How do the differ?

Reply:
I mean these attributes:
```
    # Query Result (from ChromaDB)
    DB_CHROMADB_QUERY_RESULT_IDS = "db.chromadb.query.result.ids"
    DB_CHROMADB_QUERY_RESULT_DISTANCES = "db.chromadb.query.result.distances"
    DB_CHROMADB_QUERY_RESULT_METADATA = "db.chromadb.query.result.metadata"
    DB_CHROMADB_QUERY_RESULT_DOCUMENTS = "db.chromadb.query.result.documents"

    # Query Result (from Pinecone)
    DB_PINECONE_QUERY_RESULT_ID = "db.pinecone.query.result.id"
    DB_PINECONE_QUERY_RESULT_SCORE = "db.pinecone.query.result.score"
    DB_PINECONE_QUERY_RESULT_VECTOR = "db.pinecone.query.result.vector"

```

In summary, ChromaDB returns attributes of type list, and Pinecone returns a different type per each attribute (str, float, list).

Reply:
We could take some actions to reduce the specificity, e.g., make pinecone also use lists. Then we can reuse some fields (ids, metadata).

In theory, we could also unify score / distance somehow, but it might be misleading, as score is the complement of the distance, I think this is the rule we could apply to pinecone:

`distance = 1 - score`

Or this to ChromaDB:

`score = 1 - distance`

But it can be confusing as it’s then a derived metric instead of a raw one.

Reply:
I'd keep the difference between the `distance` and `score`, but I like the rest.

Reply:
I mean, for consistency reasons we could always use type list here. Even for attributes that are only present in a specific DB. What do you think?

Reply:
Maybe I'm missing something. Can you explain what does each attribute mean in each context?

Reply:
Hey, @nirga sorry for the delay answering you. I needed a short break this topic :)

Let's go over this in detail we have these attributes in chroma:

```
    DB_CHROMADB_QUERY_RESULT_IDS = "db.chromadb.query.result.ids"
    DB_CHROMADB_QUERY_RESULT_DISTANCES = "db.chromadb.query.result.distances"
    DB_CHROMADB_QUERY_RESULT_METADATA = "db.chromadb.query.result.metadata"
    DB_CHROMADB_QUERY_RESULT_DOCUMENTS = "db.chromadb.query.result.documents"
```

The response format of Chroma comes as an object of lists
```
{
   ids: [1, 2, 3...],
   distances: [0.3, 0.5, 0.6...],
   metadata: ["some metadata text", "another metadata text",...],
   documents: ["retrieved text", "retrieved text2", ...]
}
```
Therefore all these fields are stored as lists. Either of numbers or strings. 

ID represents the stored document ID in ChromaDB.
Distance is how close the retrieved document is assessed to be to the query text.
Metadata is well, metadata.
And document is what the user originally stored beforehand in the Database.

Now let's take a look at Pinecone:
```
    # Query Result (from Pinecone)
    DB_PINECONE_QUERY_RESULT_ID = "db.pinecone.query.result.id"
    DB_PINECONE_QUERY_RESULT_SCORE = "db.pinecone.query.result.score"
    DB_PINECONE_QUERY_RESULT_VECTOR = "db.pinecone.query.result.vector"
```

This would come like this:

```
[
    {"id": 1, "score": 0.9, vector: [0.1, 0.2, 0.3, 0.9, -0.3,...]} "metadata": {"field2": "value1"},
    {"id": 2, "score": 0.8, vector: [0.2, 0.1, 0.4, 0.7, -0.4,...]}, "metadata": {"field3": "value2"},
    {"id": 3, "score": 0.7, vector: [0.3, 0.0, -0.3, -0.9, 0.3,...], "metadata": {"field1": "value1"}},
]
```
As you can see, ID is basically the same thing as in ChromaDB, but the query comes back in slightly different format.
The way I built the original Pinecone PR is to follow the original response structure from Pinecone query result. What we could do instead, is force the trace to follow a specific convention, e.g., make Pinecone trace result look a bit like ChromaDB result.

For instance, if we convert Pinecone result into Chroma format:
```
{
   ids: [1, 2, 3...],
   score: [0.9, 0.8, 0.6...],
   metadata: ["{\"field2\": \"value1\"}",  {\"field3\": \"value2\"},...],
   vectors: [[0.1, 0.2, 0.3, 0.9, -0.3,...], [0.2, 0.1, 0.4, 0.7, -0.4,...], [0.3, 0.0, -0.3, -0.9, 0.3,...]]
}
```

We could also go into the other direction and make Chroma look like Pinecone:

```
[
    {"id": 1, "distance": 0.3,  "document": "retrieved text", "metadata": "some metadata text",
    {"id": 2, "distance" 0.5, , "document": "retrieved text2": "another metadata text",
    {"id": 3, "distance": 0.6, "document": ..., "metadata": ...
]
```
I don't know if that helps or makes things more confusing for users! But if we do this, then we can reuse more fields across different databases. We just need to define which is the canonical format.

Reply:
Forgot to explain score and vectors attributes from Pinecone, but I think these should be clear by now.

Score is the complement of distance, e.g, how good the query matches the document, but here higher is better.
And vectors are the resulting vectors. These should be the same idea across databases, the only difference here is how the result is presented (again, a list in ChromaDB, or an attribute in Pinecone).